src/models/stores/configuration-manager.ts

```diff
@@ -39,6 +39,18 @@ export class ConfigurationManager implements UnitConfiguration {
     return this.getProp<UC["defaultProblemOrdinal"]>("defaultProblemOrdinal");
   }
 
+  get showCommentTag(){
+    return this.getProp<UC["showCommentTag"]>("showCommentTag");
+  }
+
+  get defaultTag(){
+    return this.getProp<UC["defaultTag"]>("defaultTag");
+  }
+
+  get commentTags(){
+    return  (this.showCommentTag) ? this.getProp<UC["commentTags"]>("commentTags") : [];
```

Comment:
Should the fallback here be `{}` instead of `[]`?

Reply:
Changed this line to `{}` but just got off a 1:1 with Leslie and she said she expects the order to be the order in which she writes into the `commentTags` object in the JSON file. So currently it is determined by `Object.keys(commentTags) `. So I will leave it as is.

Reply:
The reason @tealefristoe and I suggested an array for order in addition to the map in the JSON is that historically JavaScript hasn't guaranteed the iteration order of `Object.keys()`, so there was no guarantee that the order returned by `Object.keys()` would be the same as the order in the map in the JSON. I just looked at this again, however, and as of ES2020 `Object.keys()` is required to return the property keys in order of creation, so as long as the JSON parser creates the keys in the order they're encountered when the JSON is parsed, which seems like a reasonable assumption. So at this point it looks like we don't need a separate array, but before ES2020 it would have been required to guarantee the expected order.